content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,296 @@ weight: 50
 
     {{< text bash >}}
-    $ @install/tools/setupMeshEx.sh@ machineCerts ACCOUNT NAMESPACE
+
+    $ sudo systemctl start istio-auth-node-agent
+    $ sudo systemctl start istio
+
     {{< /text >}}
 
-    The generated files (`key.pem`, `root-cert.pem`, `cert-chain.pem`) must be copied to /etc/certs on each machine, readable by istio-proxy.
+After setup, the machine can access services running in the Kubernetes cluster
+or on other mesh expansion machines.
 
-*   Install Istio Debian files and start 'istio' and 'istio-auth-node-agent' services.
-Get the debian packages from [GitHub releases](https://github.com/istio/istio/releases) or:
+The following example shows accessing a service running in the Kubernetes cluster from a mesh expansion VM using
+`/etc/hosts/`, in this case using a service from the [Bookinfo example](docs/examples/bookinfo/).
+
+1.  First, on the cluster admin machine get the virtual IP address (`clusterIP`) for the service:
 
     {{< text bash >}}
-    $ source istio.VERSION # defines version and URLs env var
-    $ curl -L ${PILOT_DEBIAN_URL}/istio-sidecar.deb > istio-sidecar.deb
-    $ dpkg -i istio-sidecar.deb
-    $ systemctl start istio
-    $ systemctl start istio-auth-node-agent
+
+$ kubectl -n bookinfo get svc productpage -o jsonpath='{.spec.clusterIP}'
+10.55.246.247
+
     {{< /text >}}
 
------- Manual setup steps end ------
+1.  Then on the mesh expansion machine, add the service name and address to its `etc/hosts` file. You can then connect to
+    the cluster service from the VM, as in the example below:
 
-After setup, the machine should be able to access services running in the Kubernetes cluster
-or other mesh expansion machines.
+    {{< text bash >}}
 
-{{< text bash >}}
+$ sudo echo "10.55.246.247 productpage.bookinfo.svc.cluster.local" >> /etc/hosts
 $ curl productpage.bookinfo.svc.cluster.local:9080
 ... html content ...
-{{< /text >}}
 
-Check that the processes are running:
+    {{< /text >}}
 
-{{< text bash >}}
-$ ps aux |grep istio
-root      6941  0.0  0.2  75392 16820 ?        Ssl  21:32   0:00 /usr/local/istio/bin/node_agent --logtostderr
-root      6955  0.0  0.0  49344  3048 ?        Ss   21:32   0:00 su -s /bin/bash -c INSTANCE_IP=10.150.0.5 POD_NAME=demo-vm-1 POD_NAMESPACE=default exec /usr/local/bin/pilot-agent proxy > /var/log/istio/istio.log istio-proxy
-istio-p+  7016  0.0  0.1 215172 12096 ?        Ssl  21:32   0:00 /usr/local/bin/pilot-agent proxy
-istio-p+  7094  4.0  0.3  69540 24800 ?        Sl   21:32   0:37 /usr/local/bin/envoy -c /etc/istio/proxy/envoy-rev1.json --restart-epoch 1 --drain-time-s 2 --parent-shutdown-time-s 3 --service-cluster istio-proxy --service-node sidecar~10.150.0.5~demo-vm-1.default~default.svc.cluster.local
-{{< /text >}}
+## Running services on a mesh expansion machine
+
+You add VM services to the mesh by configuring a
+[`ServiceEntry`](/docs/reference/config/istio.networking.v1alpha3/#ServiceEntry). A service entry lets you manually add
```

Comment:
No, because that's really repetitive ("configuring a configuration"). You need to configure a "one of these things we're pointing you to the reference docs for", that's already perfectly clear.